src/zkevm_specs/evm/table.py

```diff
@@ -283,20 +276,22 @@ class Tables:
     # Each row in RWTable contains:
     # - rw_counter
     # - is_write
-    # - tag
-    # - value1
-    # - value2
-    # - value3
-    # - value4
-    # - value5
-    rw_table: Set[Array8]
+    # - key1 (tag)
+    # - key2
+    # - key3
+    # - key4
+    # - value
+    # - value_prev
```

Comment:
The table in the note is for State circuit itself. When it outputs to `rw_table`, it can get access to `value_prev` by rotation directly instead of having it in witness in the same row.

Reply:
make sense Thx!